grpc/src/main/java/com/avast/metrics/grpc/TimersCache.java

```diff
@@ -0,0 +1,25 @@
+package com.avast.metrics.grpc;
+
+import com.avast.metrics.api.Monitor;
+import com.avast.metrics.api.Timer;
+
+import java.util.concurrent.ConcurrentHashMap;
+
+class TimersCache {
+    private final Monitor monitor;
+    private final ConcurrentHashMap<String, Timer> timers = new ConcurrentHashMap<>();
+
+    public TimersCache(final Monitor monitor) {
+        this.monitor = monitor;
+    }
+
+    public Timer get(String name) {
```

Comment:
This method could be IMHO implemented as `return timers.computeIfAbsent(name, monitor::newTimer);`

The `synchronized` is not definitely required, and it's better to access the `ConcurrentHashMap` less times.

Reply:
You are right. I misunderstood the doc of the method. I read it again and it's clear now :-)
Fixed